src/local/document.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+
+// The local Cozy Desktop metadata, as stored in PouchDB
+export type LocalDoc = {
+  _id: string,
+  // TODO: v3: Rename to md5sum to match remote
+  checksum?: ?string,
```

Comment:
Should probably have been `checksum?: string`, so we have either a key with a non-null value, or no key at all.

But if checksums became mandatory, maybe we should just remove the mobile-specific test and corresponding code, and enforce it in the flow declaration.